### ISSUE TITLE
איך נפתחות רשומות זבל בטבלת משתמשים

### ISSUE THREAD

**Issue by natan1957** [natan1957/mymentor-portal#335]
![image](https://user-images.githubusercontent.com/6491631/51090928-3e530800-178c-11e9-981e-ed933790c5cf.png)

**Comment by igoro1975** [comment]
מעניין מאוד

**Comment by natan1957** [comment]
כן, גם אותי :)  צריך לעשות שם איזה מלכודת.. כך שבהתווספות כזו שורה ידווח מי
מה ולמה

**Comment by igoro1975** [comment]
אוסיף לוגים

**Comment by igoro1975** [comment]
טיפלתי.... נתפוס אותם :)

**Comment by natan1957** [comment]
יש לך דרך מהירה למחוק את רשומות הזבל ?
הדשבורד מרשה רק 8 בכל פעם
